pwm/asg.tf

```diff
@@ -59,10 +59,14 @@ resource "aws_autoscaling_group" "asg" {
     data.terraform_remote_state.vpc.vpc_private-subnet-az2,
     data.terraform_remote_state.vpc.vpc_private-subnet-az3
   )}"]
+  enabled_metrics           = [
+    "GroupMinSize", "GroupMaxSize", "GroupDesiredCapacity", "GroupInServiceInstances", "GroupPendingInstances",
+    "GroupStandbyInstances", "GroupTerminatingInstances", "GroupTotalInstances"
+  ]
   launch_configuration      = "${aws_launch_configuration.launch_cfg.id}"
-  min_size                  = "1"
-  max_size                  = "10"
-  desired_capacity          = "${var.pwm_config["desired_count"]}"
+  min_size                  = 3
+  desired_capacity          = 3
+  max_size                  = 30
```

Comment:
I'd say these would be better as env vars as you probably don't want them scaling to the same degree in sandpit as prod